crates/primitives/src/revm/env.rs

```diff
@@ -184,15 +184,19 @@ pub fn fill_tx_env_with_beacon_root_contract_call(env: &mut Env, parent_beacon_b
 }
 
 /// Fill transaction environment from [TransactionSignedEcRecovered].
+#[cfg(not(feature = "optimism"))]
+pub fn fill_tx_env_with_recovered(tx_env: &mut TxEnv, transaction: &TransactionSignedEcRecovered) {
+    #[cfg(not(feature = "optimism"))]
```

Comment:
I think this should be superseded by the function scoped attribute macro
```suggestion
```

Reply:
oops!